app/presenters/candidate_interface/application_form_presenter.rb

```diff
@@ -36,7 +36,7 @@ def sections_with_validations
     def sections_with_completion
       [
         # "Courses" section
-        [:course_choices, course_choices_completed?],
+        ([:course_choices, course_choices_completed?] if RecruitmentCycle.current_year < 2024),
```

Comment:
Minor as we'll look to remove this field but I think there's a `CONTINUOUS_APPLICATIONS_CYCLE_YEAR` constant

Reply:
Yeah I didn't want to get into this class. It looks like it needs revisiting. I can add the const